doc/man-sections/client-options.rst

```diff
@@ -154,6 +154,67 @@ configuration.
 --connect-timeout n
   See ``--server-poll-timeout``.
 
+--dns args
+  Client DNS configuration to be used with the connection.
+
+  Valid syntaxes:
+  ::
+
+      dns search-domains domains
+      dns server n option parameters
+
+  The dns ``--dns search-domains`` directive takes one or more domain names
+  to be added as DNS domain suffixes. If it is repeated multiple times within
+  a configuration the domains are appended, thus e.g. domain names pushed by
+  a server will ammend locally defined ones.
+
+  The ``--dns server`` directive is used to configure DNS server ``n``.
+  The server id ``n`` must be a value between -128 and 127. For pushed
+  DNS server options it must be between 0 and 127. The server id is used
+  to group options and also for ordering the list of configured DNS servers;
+  lower numbers come first. DNS servers being pushed to a client replace
+  already configured DNS servers with the same server id.
+
+  Valid options for ``--dns server`` are:
+  ::
+
+     dns server n address addr[:port] [addr[:port]]
```

Comment:
Right, I'll move them. Originally I had this separate because server options are at a different level, but users won't care and it is better to have them all together for better overview.